src/m365/tenant/commands/service/service-message-list.ts

```diff
@@ -0,0 +1,96 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption } from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  workload: string;
+}
+
+class TenantServiceMessageListCommand extends SpoCommand {
```

Comment:
This command is not related to SharePoint and it shouldn't inherit from the `SpoCommand` base class but rather from the generic `Command` class